src/org/etools/j1939_84/controllers/part02/Part02Step17Controller.java

```diff
@@ -240,9 +240,9 @@ protected void run() throws Throwable {
 
                 if (!notAvailableSPNs.isEmpty()) {
                     // 6.2.17.5.b. If no response/no valid data for any SPN requested in 6.2.16.3.a, send global message
-                    // to request that SPN(s).
-                    // Re-request the missing SPNs globally
-                    String globalMessage = "Global Request for PGN " + pgn + " for SPNs " + String.join(", ",
+                    // to request that SPN(s). Re-request the missing SPNs globally
+                    String globalMessage = "Global Request required for PGN " + pgn + " for broadcast SPNs "
```

Comment:
This needs to be split into two message.  The message is also displayed to in the report and that message should remain unchanged.